app.js

```diff
@@ -39,6 +42,30 @@ client.on('message', (wat, tags, message, self) => {
     users[tags.username] = true;
     // display current count page.
     countElement.textContent = Object.keys(users).length;
+    if (!viewers == 0) percentElement.textContent = ((Object.keys(users).length / viewers) * 100).toFixed(2) + '%';
+    else percentElement.textContent = '0%';
     usersElement.textContent = Object.keys(users).join(', ');
   }
 });
+const headers = {
+  'Client-ID': '91rougkc35nq63ygvxrix9kxzno4y9',
```

Comment:
Is this client id something that should be kept secret? e.g. can it be used for things that could potentially be dangerous?

Reply:
Nope, its completely fine (I made 100% sure of this before I put it on github)! https://discuss.dev.twitch.tv/t/should-you-expose-your-client-id/14498/5

Reply:
Lets move this to a config file though, so I can easily set my own client id. You can see an example of how we've done this here: https://github.com/CodingGarden/stream-overlay-alerts/blob/master/js/config.example.js